conda/recipes/cugraph-pyg/meta.yaml

```diff
@@ -29,13 +29,14 @@ requirements:
     - distributed >=2023.1.1
     - numba >=0.56.2
     - numpy
-    - pytorch <=1.12.1
-    # - pyg (uncomment once pyg 2.2 is released)
+    - pytorch >=2.0
+    - cupy >=9.5.0,<12.0.0a0
+    - cugraph ={{ version }}
+    - pyg-nightly
```

Comment:
just want to confirm that we're putting a nightly dependency in our release packages, or are we planning on changing this before code freeze?  If not, can we put an upper-bound pin on this so users installing this package some time later don't pull in a possibly incompatible newer nightly version?

Reply:
I just changed this now to resolve the issue of pulling the wrong version of cugraph-pyg.  We will need to bind to pyg 2.3 once it is released, which is supposed to be next week.